src/wp-includes/cache-compat.php

```diff
@@ -35,3 +35,21 @@ function wp_cache_get_multiple( $keys, $group = '', $force = false ) {
 		return $values;
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_flush_runtime' ) ) :
+	/**
+	 * Removes all cache items, from the in-memory runtime cache.
+	 *
+	 * Compat function to mimic wp_cache_flush_runtime().
+	 *
+	 * @ignore
+	 * @since 6.0.0
+	 *
+	 * @see wp_cache_flush_runtime()
+	 *
+	 * @return bool False on failure, true on success
+	 */
+	function wp_cache_flush_runtime() {
+		return wp_cache_flush();
```

Comment:
Supposing an external object cache is in use which does not yet support `wp_cache_flush_runtime()` itself: another plugin or library calling this function will inadvertently flush the persistent cache (even though it presumably called this function explicitly wishing to avoid that outcome).

Would there be value to something like:

```suggestion
		return wp_using_ext_object_cache() ? false : wp_cache_flush();
```

(That is, turn this back-fill into a no-op when an external object cache is in use?)